concent_api/api_testing_common.py

```diff
@@ -269,7 +269,7 @@ def _is_a_test(x):
 def execute_tests(tests_to_execute, objects, **kwargs):
     tests = [objects[name] for name in tests_to_execute]
     for test in tests:
-        test_id = kwargs['test_id'] + test.__name__
+        test_id = kwargs['test_id'] + ''.join(test.__name__.split('_'))
```

Comment:
maybe `'0'.join(test.__name__.split('_'))`? `test_id` would be easier to read ;)

Reply:
I have changed it for quick merge